index.html

```diff
@@ -10,28 +10,109 @@
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 <body>
-   <header>
-      <a class="logo" href="index.html"> <img  src="resources/image_geometry_1.svg" alt="Mosaic above"> </a>
-      <nav>
-          <a id="menu1" href="index.html"> <img src="resources/ic_menu.svg" alt="Menu"></a>
-      </nav>
-  </header>
-  <section>
-   <div id="Title">
-      <h1>Ivan Martinez</h1>
-   </div>   
-   <div id="legend">
-      <p> “Computers are incredibly fast, accurate, and stupid. Human beings are incredibly slow, inaccurate, and brilliant. Together they are powerful beyond imagination.”  
-          Albert Einstein..</p>
-   </div>
+   <section class="sec-portada">
+      <header> 
+         <a class="logo" href="index.html"> <img  src="resources/image_geometry_1.svg" alt="Mosaic above"> </a>
+         <nav>
+            <i class="menu-hamburg fa fa-bars"></i>
+         </nav>
+      </header>
+      <h1 class="title">Ivan Martinez</h1>   
+      <p class="parrafo"> “Computers are incredibly fast, accurate, and stupid. Human beings are incredibly slow, inaccurate, and brilliant. Together they are powerful beyond imagination.”  
+            Albert Einstein..</p>
    
-   <button>Start collaboration</button>
+      <button>Start collaboration</button>
 
-   <footer>
-       <img class="img2" src="resources/image_geometry_2.svg" alt="Mosaic bottom">
-   </footer>
-</section>     
+      <footer>
+         <img class="img2" src="resources/image_geometry_2.svg" alt="Mosaic bottom">
+      </footer>
+      <h1 class="tit-projects">Projects</h1>
+   </section>
+
+   <section class="sec-works">
+      <article class="project">
+         <img src="resources/P1-IMG.png" alt="imagen projecto 1">
+         <div class="text-content">
+            <div class="pr-description">
+               <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                  sed do eiusmod tempor incididunt ut labore et dolore magna 
+                  aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                   ullamco laboris nisi ut aliquip ea eiusmod.</p>               </div>
```

Comment:
- [ ] At the moment we don't need `Lorem Ipsum` we will use it later for the pop-up window. Kindly use the title `Project name goes here` instead. 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/185765985-10151159-4701-4e02-a7d6-98b3fa0f3cf3.png)